pages/index.tsx

```diff
@@ -9,7 +9,7 @@ export default function Home() {
     <>
       <Head>
         <title>SLE Base Container Images</title>
-        <meta name='description' content='Minimal SUSE Linux Enterprise Server 15-based images that you can use to develop, deploy, and share applications' />
+        <meta name='description' content='Secure software innovation' />
```

Comment:
Why this change? Previous text appeared more matching for me?

Reply:
From a copy perspective, the original text was very verbose. We need a maximum of three/four words for this section since this is like a tagline of-sorts